src/backend/api/reviews.js

```diff
@@ -0,0 +1,117 @@
+const express = require("express");
+const { resolve } = require("path");
+const router = express.Router();
+const knex = require("../database");
+
+// 1./api/reviews	GET	Returns all reviews.
+router.get("/", async (request, response) => {
+  try {
+    // knex syntax for selecting things. Look up the documentation for knex for further info
+    const title = await knex("Review").select("*");
+    if(title.length !== 0){
+      response.send(title);
+    }else {
+      response.status(404).json({Message:"No Reviews Found"});
+    }    
+  } catch (error) {
+    throw error;
+  }
+});
+
+// 2./api/meals/:meal_id/reviews	GET	Returns all reviews for a specific meal
+router.get("/:meal_id/reviews", async (request, response) => {
```

Comment:
I believe the path to this route will become `GET /api/reviews/:meal_id/reviews`. To resolve this you should move this route to the router in `meals.js`.

Reply:
This endpoint mentioned under api/reviews router .

Reply:
Yes but the route should be moved to the meals router so the route becomes `GET /api/meals/:meal_id/reviews`. Otherwise you can have a request like `GET /api/reviews/1/reviews` which doesn't look right and it's unclear what it does.